.github/workflows/scp-deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Run SCP deploy
+
+on:
+  workflow_dispatch:
+
+jobs:
+  scp-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          java-version: 21
+          distribution: 'temurin'
+          server-id: jfrog-central
+          server-username: INTERNAL_USERNAME
+          server-password: INTERNAL_PASSWORD
+          cache: maven
+      - name: Run Maven Package Step
+        run: |
+          mvn -B -U package -Dmaven.test.skip=true -DuseInternalRepo=true
+        env:
+          INTERNAL_USERNAME: ${{ secrets.JFROG_USERNAME }}
+          INTERNAL_PASSWORD: ${{ secrets.JFROG_PASSWORD }}
+      - name: Set up SSH key
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.SCP_CERTIFICATE }}" > ~/.ssh/id_rsa
```

Comment:
@zcgandcomp Please configure the following secrets.

secrets.SCP_CERTIFICATE
secrets.SCP_HOST
secrets.SCP_USERNAME